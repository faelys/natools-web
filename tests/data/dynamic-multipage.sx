(""
   (elements
      (main-text
         "<p>This is a dynamic multipage element, titled \""
         (element title)
         "\".</p>\n"
         (element multipage-text "")
         (render page-comments)))
   (comment-list
      (tags comments)
      (filter comment-filter)
      (backend transient-comments))
   (comment-path-prefix dynamic-multipage))
("+"
   (elements
      (title "Multipage index")
      (main-text
         "<p>This is a dynamic multipage index.</p>\n"
         (tags pages/dyn-fragments (all-elements (template
            "<h2>" (element title) "</h2>\n"
            (element multipage-text "")))))))
("##part1"
   (elements
      (title "Part one"))
   (tags (dp1 pages/dyn-fragments)))
("##part2"
   (elements
      (title "Part two"))
   (tags (dp2 pages/dyn-fragments)))
("+/subpage1"
   (elements
      (title "Subpage one"))
   (tags (ds1 pages/dyn-fragments))
   (comments))
("+/subpage2"
   (elements
      (title "Subpage two"))
   (tags (ds2 pages/dyn-fragments))
   (comments -002))
