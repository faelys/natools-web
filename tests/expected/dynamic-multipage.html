<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Multipage index</title>
  </head>
  <body>
<h1>Multipage index</h1>
<p>This is a dynamic multipage index.</p>
<h2>Part one</h2>
<h2>Part two</h2>
<h2>Subpage one</h2>
<h2>Subpage two</h2>
    <div class="sidebar">
<h1>Side bar</h1>
    </div>
    <div class="footer">
<p>Default footer: <a href="/index">return to index</a></p>
    </div>
  </body>
</html>
